content/post/against-ab-tests.md

```diff
@@ -0,0 +1,93 @@
+---
+author: Michael Kaminsky
+title: "Against A/B Tests"
+draft: True
+publishDate: "2018-12-01"
+tags:
+ - Statistics
+ - Testing
+ - Web Analytics
+---
+
+The notion of an A/B test is premised on the fundamentally flawed assumption that there is one version of some treatment that is better for all users at all times. Analytics practitioners should embrace this variation and start designing systems that allow for (and encourage) non-binary outcomes of tests.
```

Comment:
The second sentence is slightly awkward following the first. It sounds like you're about to set up an argument against the notion of "one size fits all". Then in the second sentence you use the positive affirmation `should embrace` which causes me to double take on what exactly I should be embracing, which is actually `variation`, *not* the subject of the first sentence, which is lack of variation. This distinction could be more explicit.

Reply:
Great comment. I switched to 

>Analytics practitioners should reject the assumptions of homogeneity and start designing systems that allow for (and encourage) non-binary outcomes of tests.

Which I think is better because it avoids the polarity switch, but don't think it's perfect. any ideas?